src/mapping/Mapping.hpp

```diff
@@ -21,12 +21,14 @@ class Mapping {
    * then scales the mapped such that the integrals on both sides of the interface
    * are equal. Values integrated over some area should be mapped conservative or
    * scaled-consistent, while area independent values such as pressure or stresses
-   * should be mapped consistent.
+   * should be mapped consistent. There is also a volume version 
+   * where volume integrals are preserved instead of surface integrals.
```

Comment:
The docs seem a bit outdated. They are not incorrect, but I guess you could immediately merge the volume stuff into the scaled consistent description?!

Reply:
Updated this. Can you review the new version ?